pkg/docker/docker_test.go

```diff
@@ -1,22 +0,0 @@
-package docker
```

Comment:
Why delete this?

Reply:
undelete, and fix injecting `Tag` problem in test by building and testing against `:latest`